### PR TITLE
Add repositories and pluginRepositories to pom.xml

### PR DESCRIPTION
**Pull request by rvanheest**
@wilkos-dans and I came to this after it did not work for him. @janvanmansum, you forgot to add that this morning!

### CODE REVIEW

**Comment by rvanheest** [comment]
To be honest, I don't think we should actually add the Dans dependencies. This also requires to add the mirror to the `.m2/settings.xml`

**Comment by rvanheest** [comment]
Could you also add this project to Travis to avoid problems like this in the future?

File: pom.xml [hunk 1]
```diff
@@ -86,6 +86,24 @@
             <version>1.1.1</version>
         </dependency>
     </dependencies>
+    <repositories>
```
Comment (janvanmansum) [inline pom.xml:89]: Good point, only it should be:
```
 <repositories>
        <repository>
            <id>DANS</id>
            <releases>
                <enabled>true</enabled>
            </releases>
            <url>http://maven.dans.knaw.nl/</url>
        </repository>
    </repositories>
    <pluginRepositories>
        <pluginRepository>
            <id>DANS</id>
            <releases>
                <enabled>true</enabled>
            </releases>
            <url>http://maven.dans.knaw.nl/</url>
        </pluginRepository>
    </pluginRepositories>
```
Those are accessible from outside the DANS network.

**Review by janvanmansum — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (janvanmansum) [reply pom.xml:89]: While you are at it you might also add the travis config.

**Comment by rvanheest** [comment]
Closing this PR for now. Moved these changes to the other PR at #21.
@janvanmansum I added the travis link in #21, but I think you have to add it to travis yourself. I don't have those rights, if I remember correctly.

**Pull request closed without merging**